keyboards/kbdfans/kbd75/keymaps/kingwangwong/keymap.c

```diff
@@ -0,0 +1,144 @@
+enum layer_names {
```

Comment:
```suggestion
#include QMK_KEYBOARD_H

enum layer_names {
```

This should fix the Travis errors.